feedback/management/commands/export_feedback.py

```diff
@@ -49,7 +49,7 @@ def __iter__(self):
             yield {
                 'uid': data.id,
                 'student': data.student_id,
-                'tags': list(map(lambda t: str(t), data.tags.all())),
+                'tags': list(map(lambda t: str(t), data.tags.all())), # pylint: disable=unnecessary-lambda
```

Comment:
FIXME later: the lambda could be replaced with `str` directly. It is not necessary to wrap `str` inside the lambda.